### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by trayo**

### CODE REVIEW

File: brave/src/main/java/brave/sampler/CountingSampler.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 /**
  * This sampler is appropriate for low-traffic instrumentation (ex servers that each receive <100K
- * requests), or those who do not provision random trace ids. It not appropriate for collectors as
+ * requests), or those who do not provision random trace ids. It is not appropriate for collectors as
```
Comment (devinsba) [inline brave/src/main/java/brave/sampler/CountingSampler.java]: It looks like our line lengths were fairly normal, can you move the `as` down on the next line.
```suggestion
 * requests), or those who do not provision random trace ids. It is not appropriate for collectors
 * as the sampling decision isn't idempotent (consistent based on trace id).
```

File: brave/src/main/java/brave/sampler/CountingSampler.java [hunk 2]
```diff
@@ -5,7 +5,8 @@
 
 /**
  * This sampler is appropriate for low-traffic instrumentation (ex servers that each receive <100K
- * requests), or those who do not provision random trace ids. It not appropriate for collectors as
+ * requests), or those who do not provision random trace ids. It is not appropriate for collectors
+ * as the sampling decision isn't idempotent (consistent based on trace id).
  * the sampling decision isn't idempotent (consistent based on trace id).
```
Comment (devinsba) [inline brave/src/main/java/brave/sampler/CountingSampler.java]: Well, i guess that feature is in beta....
```suggestion
```

Comment (devinsba) [reply brave/src/main/java/brave/sampler/CountingSampler.java]: I wonder how to do a multiline suggestion?

Comment (trayo) [reply brave/src/main/java/brave/sampler/CountingSampler.java]: Heh, I thought I would give it a try. Seemed convenient. :)

**Comment by devinsba** [comment]
Looks like a flaky test in gRPC land is failing the build. I'll create an issue to address it. Thanks

**Pull request merged**